### DISCUSSION TITLE
Redraw gets requested more frequently when mouse is moving

### DISCUSSION THREAD

**Discussion by fazil47** [rust-windowing/winit#3963] [Q&A]
I'm on winit 0.29.15 and wgpu 0.20. Winit redraws slowly when I'm not moving the mouse, Is there a way to change that? I've tested on macOS and windows. I've tried using `ControlFlow::Poll` but that doesn't work either.

https://github.com/user-attachments/assets/031b363d-ada0-4b80-b730-06a8ef931d18

**Comment by madsmtm** [comment]
Depends on how you're rendering with Wgpu? And are you rendering inside `RedrawRequested`?

Also, does wgpu 22 exhibit the same behaviour?

**Reply by fazil47** [reply]
I am rendering inside `RedrawRequested`, here's the code: [wgpu-renderer](https://github.com/fazil47/wgpu-renderer/blob/6f1179717309eadfd79208b4bd96283b572bc00b/app/src/renderer.rs#L22-L63)

> Also, does wgpu 22 exhibit the same behaviour?

Hope to try that soon, I'm still on 0.20 because winit 0.30's changes make WASM difficult.
